CameraXBasic/app/src/main/res/navigation/nav_graph.xml

```diff
@@ -17,8 +17,8 @@
 <navigation
     xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
-    android:id="@+id/nav_graph"
-    app:startDestination="@id/permissions_fragment">
+    app:startDestination="@id/permissions_fragment"
+    android:id="@+id/nav_graph">
```

Comment:
If you change the order back to what it was, there will be no changes to this file :)

Reply:
Have reverted that. While I could provide two screenshots comparing 3x "preview unavailable" with a XML preview, which shows the content of all the fragments in the navigation editor. This would be a case for a separate PR to be made.